packages/shell-evaluator/package.json

```diff
@@ -26,7 +26,7 @@
     "node": ">=12.4.0"
   },
   "dependencies": {
-    "@mongosh/async-rewriter": "0.0.0-dev.0",
```

Comment:
The previous dependency here was unused. If we move forward with this, we can eventually remove any async rewriting dependency from `@mongosh/shell-api` altogether.